build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -41,16 +48,33 @@
 
   - name: Create a Secret for the dockercfg
     ansible.builtin.command: oc create secret generic -n {{ namespace }} service-telemetry-framework-index-dockercfg --from-file=.dockerconfigjson=working/service-telemetry-framework-index/config.json --type=kubernetes.io/dockerconfigjson
```

Comment:
Should these `command` calls get updated to a consistent usage since there are changes here to `command.cmd` syntax?

Reply:
Using command.cmd is needed if we have multiple arguments passed to the modules e.g. chdir, creates, etc

ansible lint may give out about this, if using `command: oc ...` might be considered free-form, which the linter discourages.